apps/widminbt/widget.js

```diff
@@ -1,15 +1,13 @@
-(()=> {
-  WIDGETS.minbt={area:"tr",width:NRF.getSecurityStatus().connected?0:15,draw:function() {
-    if(this.width<15)return;
+WIDGETS["bluetooth"] = {area: "tr", width: NRF.getSecurityStatus().connected ? 0 : 15, draw: function () {
+    if (WIDGETS.bluetooth.width < 15) return;
```

Comment:
You could keep `this` here if you prefer:
```suggestion
    if (this.width < 15) return;
```

Reply:
I used this example to mimic same way widbthide is using.
In fact I found widbthide is exact opposite of widminbt. 
It seems nice to keep it simple for both so one way or another seems fit but changes needs to be done on both apps.